src/components/Visualizations/Table/KDA.jsx

```diff
@@ -20,7 +20,7 @@ const KDA = ({ kills, deaths, assists, matchId = uuid.v4() }) => {
           <div style={{ width: `${(assists * 100) / kdaSum}%`, backgroundColor: palette.green }} />
         </div>
         <ReactTooltip id={`kda-${matchId}`} place="right" type="light" effect="float">
-          {`${strings.abbr_kda}: ${Number(((kills + assists) / Math.max(deaths, 0)).toFixed(2))}`}
+          {`${strings.abbr_kda}: ${Number(((kills + assists) / Math.max(deaths, 1)).toFixed(2))}`}
```

Comment:
We can update the code to compute future values, but back-computing the old data will be very expensive.